analytics/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
```

Comment:
This is strange, I changed a file in analytics? Wow!

Reply:
Yes, it's a bit strange indeed